containers/ddev-webserver/Dockerfile

```
@@ -16,6 +16,7 @@ ENV CAROOT=/mnt/ddev-global-cache/mkcert
 # BUILDPLATFORM is the platform of the build host (e.g. linux/amd64)
 ARG TARGETPLATFORM
 ARG BUILDPLATFORM
+ENV DEFAULT_LOCALES="en_CA.UTF-8 UTF-8\nen_US.UTF-8 UTF-8\nen_GB.UTF-8 UTF-8\nde_DE.UTF-8 UTF-8\nde_AT.UTF-8 UTF-8\nfr_CA.UTF-8 UTF-8\nfr_FR.UTF-8 UTF-8\nja_JP.UTF-8 UTF-8\nru_RU.UTF-8 UTF-8"
```

I don't see `$DEFAULT_LOCALES` in the resulting image (why? maybe `\n` is to blame), but let's use `ARG` to be sure. And I added a new line at the end, so `cat /etc/locale.gen` looks better.

```suggestion
ARG DEFAULT_LOCALES="en_CA.UTF-8 UTF-8\nen_US.UTF-8 UTF-8\nen_GB.UTF-8 UTF-8\nde_DE.UTF-8 UTF-8\nde_AT.UTF-8 UTF-8\nfr_CA.UTF-8 UTF-8\nfr_FR.UTF-8 UTF-8\nja_JP.UTF-8 UTF-8\nru_RU.UTF-8 UTF-8\n"
```

---

I'm pretty sure the reason you don't see DEFAULT_LOCALES in the resulting image is that it's from a different stage. It's an ENV there in ddev-webserver-base, but the final build is ddev-webserver, 2 stages later. ENVs don't survive those stages. 

The `\n` is actually useful here because I did a printf instead of `echo`. The echo would have added its own.

---

I do sure like using ARG vs ENV, will eventually learn to do that :)